src/openrct2/drawing/Drawing.Sprite.cpp

```diff
@@ -618,6 +618,9 @@ void FASTCALL
     width = std::min(imgMask->width, imgColour->width);
     height = std::min(imgMask->height, imgColour->height);
 
+    // TODO: This feels "hacky". does scrCoords really HAVE to be a const?
```

Comment:
Yes it should be const. this is pretty normal to copy the value into another variable during the function.